pkg/util/k8sutil/k8sutil.go

```diff
@@ -290,6 +292,17 @@ func NewEtcdPod(m *etcdutil.Member, initialCluster []string, clusterName, state,
 }
 
 func MustNewKubeClient() kubernetes.Interface {
+	// See https://github.com/coreos/etcd-operator/issues/731#issuecomment-283804819
```

Comment:
See my reasoning at https://github.com/coreos/etcd-operator/issues/731#issuecomment-283804819

Retrying will always return that error